spark-3.5.yaml

```diff
@@ -120,7 +131,7 @@ subpackages:
                 val sum = rdd.reduce(_ + _)
                 assert(sum == 15)
                 EOF
-                cat SimpleJob.scala | /usr/lib/spark/bin/spark-shell
+                cat SimpleJob.scala | /usr/lib/spark/bin/spark-shell --conf spark.jars.ivy=/tmp/.ivy
```

Comment:
I'm not sure I understand why we need pass --conf

Reply:
@EyeCantCU because of this error `2024/11/19 00:35:51 WARN Exception in thread "main" java.lang.IllegalArgumentException: basedir must be absolute: ?/.ivy2/local name="Test spark-3.5 with OpenJDK 11" name="Run a simple Scala test script"`